policy/modules/system/systemd.te

```diff
@@ -1527,8 +1541,19 @@ seutil_libselinux_linked(systemd_user_session_type)
 allow systemd_user_runtime_dir_t self:capability { fowner chown sys_admin dac_read_search dac_override };
 allow systemd_user_runtime_dir_t self:process setfscreate;
 
+manage_sock_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_notify_t, systemd_user_runtime_notify_t)
+
```

Comment:
i might be wrong but i dont think systemd_user_runtime_dir_t needs to manage that notify sock file

Reply:
systemd_user_runtime_dir_t *does* need to be able to "delete" all of /run/user/* though.

but you can address that on a higher leve (which would make it more robust as well)l

Reply:
OK. Adding the access to remove user runtime stuff fixed this also.